### PR TITLE
feat(tests): added tests for pubdata chunk publisher

### PR DESCRIPTION
**Pull request by koloz193**
# What ❔

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

File: system-contracts/test/PubdataChunkPublisher.spec.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import { prepareEnvironment } from "./shared/mocks";
+import { deployContractOnAddress, getWallets } from "./shared/utils";
+
+describe("PubdataChunkPublisher tests", () => {
+  let wallet: Wallet;
+  let l1MessengerAccount: ethers.Signer;
+
+  let pubdataChunkPublisher: PubdataChunkPublisher;
+
+  const genRanHex = (size) => [...Array(size * 2)].map(() => Math.floor(Math.random() * 16).toString(16)).join("");
```
Comment (StanislavBreadless) [inline system-contracts/test/PubdataChunkPublisher.spec.ts:16]: ethers.utils.randomBytes

File: system-contracts/test/PubdataChunkPublisher.spec.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    });
+
+    it("Too Much Pubdata", async () => {
+      const pubdata = "0x" + genRanHex(blobSizeInBytes * maxNumberBlobs + 1);
+      await expect(
+        pubdataChunkPublisher.connect(l1MessengerAccount).chunkAndPublishPubdata(pubdata)
+      ).to.be.revertedWith("pubdata should fit in 2 blobs");
+    });
+
+    it("Publish 1 Blob", async () => {
```
Comment (StanislavBreadless) [inline system-contracts/test/PubdataChunkPublisher.spec.ts:49]: would it be possible to check the values of the system logs that have been emitted?

Comment (koloz193) [reply system-contracts/test/PubdataChunkPublisher.spec.ts:49]: talked offline, unfortunately not. we looked at the transaction and contract receipt but there are no system logs present there.

File: system-contracts/test/PubdataChunkPublisher.spec.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import { prepareEnvironment } from "./shared/mocks";
+import { deployContractOnAddress, getWallets } from "./shared/utils";
+
+describe("PubdataChunkPublisher tests", () => {
+  let wallet: Wallet;
+  let l1MessengerAccount: ethers.Signer;
+
+  let pubdataChunkPublisher: PubdataChunkPublisher;
+
+  const genRandHex = (size) => ethers.utils.hexlify(ethers.utils.randomBytes(size));
```
Comment (StanislavBreadless) [inline system-contracts/test/PubdataChunkPublisher.spec.ts:16]: I think you can just use `ethers.utils.randomBytes` everywhere. I think that ethers.js should be fine parsing array of bytes without it being a hex string

**Review by StanislavBreadless — APPROVED** [review]

**Pull request merged**